VTEX - Pricing API.json

```diff
@@ -344,6 +344,128 @@
                         "description": "OK"
                     }
                 }
+            },
+            "patch": {
+                "tags": [
+                    "Prices and Fixed Prices"
+                ],
+                "summary": "Create or update base price or fixed prices",
+                "description": "Updates the fixed prices of an SKU. A fixed price is an optional price defined for a specific trade policy or a price table. It can include a minimum quantity and an effective date range that determines when the price is active.\n\n This request does not remove existing fixed prices. Instead, it updates or adds new fixed prices without affecting trade policies or price tables that are not included in the request. If a fixed price already exists for the specified SKU and matches the activation conditions (i.e., dateRange and minQuantity), it will be replaced. Otherwise, a new fixed price will be created.\r\n\r\n\r\n\r\n## Permissions\r\n\r\nAny user or [application key](https://developers.vtex.com/docs/guides/api-authentication-using-application-keys) must have at least one of the appropriate [License Manager resources](https://help.vtex.com/en/tutorial/license-manager-resources--3q6ztrC8YynQf6rdc6euk3) to be able to successfully run this request. Otherwise they will receive a status code `403` error. These are the applicable resources for this endpoint:\r\n\r\n| **Product** | **Category** | **Resource** |\r\n| --------------- | ----------------- | ----------------- |\r\n| Pricing | Price List | **Modify prices** |\r\n\r\nThere are no applicable [predefined roles](https://help.vtex.com/en/tutorial/predefined-roles--jGDurZKJHvHJS13LnO7Dy) for this resource list. You must [create a custom role](https://help.vtex.com/en/tutorial/roles--7HKK5Uau2H6wxE1rH5oRbc#creating-a-role) and add at least one of the resources above in order to use this endpoint. To learn more about machine authentication at VTEX, see [Authentication overview](https://developers.vtex.com/docs/guides/authentication).\r\n\r\n>\u2757 To prevent integrations from having excessive permissions, consider the [best practices for managing app keys](https://help.vtex.com/en/tutorial/best-practices-application-keys--7b6nD1VMHa49aI5brlOvJm) when assigning License Manager roles to integrations.",
```

Comment:
the summary mentions "base prices" but the description doesn't. can you add a description for these concepts, like you did in the other endpoint?

Reply:
also explain what this request does to base prices, besides fixed prices